user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.templates.js

```diff
@@ -91,10 +91,10 @@ define([
           "{{#ifCond file.path '.trash'}}" +
           "<div id='{{file.objectId}}' class='trash folder' path='{{file.path}}' ext='{{file.name}}' desc='{{file.name}}'>" +
           "{{else}}" +
-              "{{#if file.title}}" +
```

Comment:
I should have been handling the title in browser.js reformatresponse and not in the template. I fixed it here as I decided to handle the element class via template conditional.